pmoves/services/gateway/gateway/api/chit.py

```
@@ -70,8 +79,7 @@ def decrypt_anchor(const: Dict[str, Any]) -> None:
     if not CHIT_DECRYPT_ANCHORS:
         raise HTTPException(status_code=400, detail="Encrypted anchor but CHIT_DECRYPT_ANCHORS=false")
     iv = base64.b64decode(enc["iv"]); salt = base64.b64decode(enc["salt"]); ct = base64.b64decode(enc["ct"])
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)